internal/controllers/clusterextension_registryv1_validation_test.go

```diff
@@ -99,13 +100,21 @@ func TestClusterExtensionRegistryV1DisallowDependencies(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			defer func() {
 				require.NoError(t, cl.DeleteAllOf(ctx, &ocv1alpha1.ClusterExtension{}))
-				require.NoError(t, cl.DeleteAllOf(ctx, &rukpakv1alpha2.BundleDeployment{}))
 			}()
-
 			fakeCatalogClient := testutil.NewFakeCatalogClient([]*catalogmetadata.Bundle{tt.bundle})
+			mockUnpacker := unpacker.(*MockUnpacker)
+			// Verify if mockUnpacker is correctly set
+			t.Log("MockUnpacker set to:", mockUnpacker)
```

Comment:
Nit: Is this necessary?

Reply:
@bentito is looking at these few nits, and will push up a small change.

Reply:
eliminated now